gpii/node_modules/lifecycleManager/src/UserLogonRequest.js

```diff
@@ -97,7 +97,15 @@ gpii.lifecycleManager.userLogonRequest.handleSuccessRequest = function (that, pa
 gpii.lifecycleManager.userLogonRequest.handleFailedRequest = function (that, userErrorEvent, err) {
     fluid.log("UserLogonRequest fails with the error: ", err);
 
-    if (!err.ignoreUserErrors) {
+    var connectionErrCode = ["ECONNREFUSED", "ETIMEDOUT", "ECONNRESET"];
```

Comment:
Put these in a public constant, together with a public checking function

Reply:
@amb26, the ways in my mind in terms of where to put these public constants and public utility functions share by multiple GPII modules:

1. within "flowManager" module such as flowManager/src/Utility.js;
2. create a separae GPII module "gpii-utility" with Utility.js in its `src/` directory.

Either way, Utility.js will be required by various GPII modules when it's needed. In this fix, these modules are "flowManager" and "lifecycleManager".

Which way would you prefer or do you have other ideas? Thanks.

Reply:
I suggest putting this constant and function in the gpii.userErrors namespace and module